src/tools/tests/goldens/generated-schemas.wasm.kt

```diff
@@ -15,6 +15,7 @@ typealias Gold_Alias = AbstractGold.Gold_Alias
 typealias Gold_Collection = AbstractGold.Foo
 typealias Gold_QCollection = AbstractGold.Gold_QCollection
 
+
```

Comment:
Is it possible you could structure the templating such that no extra whitespace is added?

Reply:
Yeah, it was already fixed, I just didn't push in time :-)

Reply:
awesome!

Reply:
(But with your other comment about always adding the @Generated tag, it might be obsolete in a moment anyway!)